packages/server/manage_container/client/manage_to_db/client.go

```diff
@@ -326,3 +328,19 @@ func (c Client) GetMatchingColumn(dataID string) (int32, error) {
 
 	return data.Meta.MatchingColumn, nil
 }
+
+func (c Client) CreateStatusFile(jobUUID string) {
+	ls.Lock(jobUUID)
+	defer ls.Unlock(jobUUID)
+	path := fmt.Sprintf("%s/%s", resultDbPath, jobUUID)
+	os.Mkdir(path, 0777)
```

Comment:
it is just comment.  

[TODO] think about permissions.

Implementation elsewhere:
https://github.com/acompany-develop/QuickMPC/blob/985df54425ca54289e762e9507d47ec62463f388/packages/server/manage_container/client/manage_to_db/client.go#L82